wa/commands/create.py

```diff
@@ -77,7 +116,90 @@ class CreateCommand(ComplexCommand):
-        #CreateWorkloadSubcommand,
-        #CreatePackageSubcommand,
+        CreateWorkloadSubcommand,
         CreateAgendaSubcommand,
+        #CreatePackageSubcommand,
     ]
+
+
+def create_workload(name, kind='basic', where='local', check_name=True, **kwargs):
+
+    if check_name:
+        if name in [wl.name for wl in pluginloader.list_plugins('workload')]:
+            raise CommandError('Workload with name "{}" already exists.'.format(name))
+
+    class_name = get_class_name(name)
+    if where == 'local':
+        workload_dir = _d(os.path.join(settings.plugins_directory, name))
+    else:
+        workload_dir = _d(os.path.join(where, name))
+
+    try:
+        create_funcs[kind](workload_dir, name, kind, class_name, **kwargs)
+    except KeyError:
+        raise CommandError('Unknown workload type: {}'.format(kind))
+
+    print 'Workload created in {}'.format(workload_dir)
+
+
+def create_template_workload(path, name, kind, class_name):
+    source_file = os.path.join(path, '__init__.py')
+    with open(source_file, 'w') as wfh:
+        wfh.write(render_template('{}_workload'.format(kind), {'name': name, 'class_name': class_name}))
+
+def create_uiautomator_template_workload(path, name, kind, class_name):
+    uiauto_path = os.path.join(path, 'uiauto')
+    create_uiauto_project(uiauto_path, name)
+    create_template_workload(path, name, kind, class_name)
+
+def create_uiauto_project(path, name):
+    package_name = 'com.arm.wa.uiauto.' + name.lower()
+
+    copy_tree(os.path.join(TEMPLATES_DIR, 'uiauto', 'uiauto_workload_template'), path)
+
+    manifest_path = os.path.join(path, 'app', 'src', 'main')
+    mainifest = os.path.join(_d(manifest_path), 'AndroidManifest.xml')
+    with open(mainifest, 'w') as wfh:
+        wfh.write(render_template(os.path.join('uiauto', 'uiauto_AndroidManifest.xml'),
+                                  {'package_name': package_name}))
+
+    build_gradle_path = os.path.join(path, 'app')
+    build_gradle = os.path.join(_d(build_gradle_path), 'build.gradle')
+    with open(build_gradle, 'w') as wfh:
+        wfh.write(render_template(os.path.join('uiauto', 'uiauto_build.gradle'),
+                                  {'package_name': package_name}))
+
+    build_script = os.path.join(path, 'build.sh')
+    with open(build_script, 'w') as wfh:
+        wfh.write(render_template(os.path.join('uiauto', 'uiauto_build_script'),
+                                  {'package_name': package_name}))
+    os.chmod(build_script, stat.S_IRWXU | stat.S_IRWXG | stat.S_IRWXO)
+
+    source_file = _f(os.path.join(path, 'app', 'src', 'main', 'java',
+                                  os.sep.join(package_name.split('.')[:-1]),
+                                  'UiAutomation.java'))
+    with open(source_file, 'w') as wfh:
+        wfh.write(render_template(os.path.join('uiauto', 'UiAutomation.java'),
+                                  {'name': name, 'package_name': package_name}))
+
+create_funcs = {
```

Comment:
Move to the top (below imports) and update indentation should be 
```

create_funcs = {
     'basic': create_template_workload,
      ....
}
```

Reply:
Updated